src/service/batched-xhr-impl.js

```diff
@@ -83,7 +83,8 @@ export class BatchedXhr extends Xhr {
    * @private
    */
   getMapKey_(input, responseType) {
-    return removeFragment(input) + responseType;
+    const absoluteUrl = resolveRelativeUrl(input, getWinOrigin(this.win));
```

Comment:
You probably want to use `getSourceOrigin()` to avoid getting the AMP cache origin here.